java/src/main/java/ai/rapids/cudf/Table.java

```diff
@@ -1820,6 +1824,30 @@ public Table filter(ColumnView mask) {
     return new Table(filter(nativeHandle, mask.getNativeView()));
   }
 
+  /**
+   * Copy rows of the current table to an output table such that duplicate rows in the key columns
+   * are ignored (i.e., only one row from the duplicate ones will be copied). These keys columns are
+   * a subset of the current table columns and their indices are specified by an input array.
+   *
+   * Currently, the output table is sorted by key columns, using stable sort. However, this is not
+   * guaranteed in the future.
+   *
+   * @param keyColumns Array of indices representing key columns from the current table.
+   * @param keepFirst If it is true, the first row in a sequence of duplicate rows will be copied.
```

Comment:
The "sequence of duplicate rows" could make someone wonder, "what if there are duplicates that aren't sequential rows in the table?"  This is exposing implementation details into the parameter description.  I'd limit this to referring to the first row with a duplicated key vs. last row with a duplicated key and don't bother with describing sequence of rows.

Reply:
Fixed that. Sorry for my language :smile: